src/features/queries.ts

```diff
@@ -208,9 +208,9 @@ export const GET_ORDER_DETAILS = gql`
     orderDetails(orderNumber: $orderNumber) {
       orderType
       status
-      harbor
-      pier
-      berth
+      place
+      section
+      area
```

Comment:
Just to confirm, did the backend change to this generic form?

Reply:
Yes, now the backend uses these fields for both berth and winter storage places